README.md

```diff
@@ -16,7 +16,7 @@ Bespot iOS SDK offers proximity events and analytics reporting to 3rd party apps
 ## Requirements
 
 - iOS 12.0+
```

Comment:
@spvagelis  Is this still the case or we now run iOS 15.0+?

Reply:
The current version of Beengo is iOS 15, so it could be changed to iOS 15.

Reply:
I was referring to the SDK. I suppose we did not updated the deployment target to 15+. So we are OK.